### PR TITLE
Enable github actions

### PR DESCRIPTION
**Pull request by ssbarnea**

### CODE REVIEW

**Comment by ssbarnea** [comment]
@BeyondEvil after merging remember to edit settings of master branch and add protection to require all github actions to be green (now only travis is "required" to pass)

File: .github/workflows/tox.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: gh
+
+on:
+  create:  # is used for publishing to PyPI and TestPyPI
```
Comment (nicoddemus) [inline .github/workflows/tox.yml:4]: I don't see any deployment being done here, are those comments meant for later or am I missing something?

Comment (BeyondEvil) [reply .github/workflows/tox.yml:4]: I think the idea is for this to run along side Travis for a while, so we'll let Travis to the deploy until we switch over. @nicoddemus 

Correct me if I'm wrong @ssbarnea

Comment (ssbarnea) [reply .github/workflows/tox.yml:4]: Yes. That is the plan. In fact code for deploying using github actions is missing from current change. We can switch to github publishing only when we are sure we want to drop travis.

Comment (nicoddemus) [reply .github/workflows/tox.yml:4]: OK, thanks folks.

**Pull request merged**